ext/panko_serializer/panko_serializer.c

```diff
@@ -24,18 +25,20 @@ void serialize_method_fields(VALUE subject,
                              VALUE str_writer,
                              SerializationDescriptor descriptor,
                              VALUE context) {
-  VALUE method_fields = descriptor->method_fields;
+  VALUE method_fields, serializer;
```

Comment:
Why is this not volatile?

Reply:
Both of them are values which I call on them `rb_gc_mark`, why volatile is necessary here?

Reply:
ok, missed that part. Should be fine.